src/resample.jl

```diff
@@ -0,0 +1,12 @@
+function resample(A::GeoArray, proj::GeoFormat, resolution::Number;
```

Comment:
Does the projection always change here? can it be a keyword arg `crs=crs(A)`?

Reply:
I guess it follows that the resolution could also default to the existing resolution, so you could just change the crs

Reply:
I think since it's resample and not reproject it might make sense to require that a resolution be specified?